site/content/en/docs/manual/advanced/serverless-tutorial.md

```diff
@@ -898,6 +898,59 @@ $ docker logs nuclio-nuclio-pth.shiyinzhang.iog
 
 </details>
 
+### Problem with running nuctl on Windows 10
+
+If you encounter the problem with running `nuctl` on Windows 10, you can do the following:
```

Comment:
Probably the sentence can be removed. Let's recommend one way to install on Windows 10